src/database.ts

```diff
@@ -87,7 +87,7 @@ class Database {
   close() {
     if (this.conn) {
       this.conn.close();
-      this.conn = null;
+      this.conn = undefined;
```

Comment:
Why `null` does not work?

Reply:
If I use `null` i need to widen the type to `SQLiteDatabase | null`.
But I may need to do that anyway because of the eslint rules.

Reply:
It now looks like
```private conn: SQLiteDatabase | null = null;```
instead of
```private conn?: SQLiteDatabase;```

Reply:
I guess I could avoid the `= null` by making it optional.

Reply:
I reverted it to `null` because of `@typescript-eslint/recommended`